Lib/slapdtest/_slapdtest.py

```diff
@@ -56,6 +56,12 @@
 
 LOCALHOST = '127.0.0.1'
 
+CI_DISABLED = set(os.environ.get('CI_DISABLED', '').split(':'))
+if 'LDAPI' in CI_DISABLED:
+    HAVE_LDAPI = False
+else:
+    HAVE_LDAPI = hasattr(socket, 'AF_UNIX')
+
```

Comment:
What if, when `CI` is set, we only looked at `CI_DISABLED`? Maybe with an early error if `LDAPI` isn't disabled but `socket.AF_UNIX` is available.
And if `CI` is unset, do the `hasattr` check.

In this case, the `hasattr` is probably robust enough, but I think starting a "avoid feature detection if `CI` is set" pattern would be beneficial.

Reply:
I'd like to run tests without LDAPI support locally, too. Can you recommend a better way to pass an option from tox without introducing more env vars?

We could move to pytest in order to use proper fixtures and options. But I don't like to do that now.

Reply:
I misunderstood the code. Sorry for the noise!